src/main/groovy/org/shipkit/internal/gradle/versionupgrade/UpgradeDownstreamPlugin.java

```diff
@@ -63,13 +64,22 @@ public void apply(final Project project) {
 
         upgradeDownstreamExtension = project.getExtensions().create("upgradeDownstream", UpgradeDownstreamExtension.class);
 
-        final Task performAllUpdates = TaskMaker.task(project, "upgradeDownstream", new Action<Task>() {
+        final Task performAllUpdates = TaskMaker.task(project, UPGRADE_DOWNSTREAM_TASK, new Action<Task>() {
             @Override
             public void execute(final Task task) {
                 task.setDescription("Performs dependency upgrade in all downstream repositories.");
             }
         });
 
+        ShipkitExecTask ciPerformReleaseTask = (ShipkitExecTask) project.getTasks().findByName(CiReleasePlugin.CI_PERFORM_RELEASE_TASK);
```

Comment:
We can also bust this plugin into 2:
 - upgrade-downstream - adds task, extension, etc.
 - ci-upgrade-downstream - "glues" upgrade-downstream with ci-release

This would offer better function decomposition and make plugins more usable in general